model/Dockerfile

```diff
@@ -7,4 +7,6 @@ RUN apt-get update && apt-get install git-lfs && git clone https://huggingface.c
 
 COPY . .
 
-CMD uvicorn main:app --host 0.0.0.0 --port $PORT
\ No newline at end of file
+EXPOSE $PORT
```

Comment:
This won't do anyting as the environment variable is only passed after it is built. To make things easier just set a default static port, you can always change it when doing the mapping from container port to machine port